pkg/bgs

```diff
@@ -0,0 +1,21 @@
+[mirrors]
+https://github.com/Gottox/bgs/archive/master.tar.gz
+
+[vars]
+filesize=4991
+sha512=30c806c097a6fdf2afd9a17a2cc1c2a27cce65df17a33921952ac68946b2a0efbcabcff00fab67013989478844e00c17899b4036bbd69dd7a110789ee4b2622c
+pkgver=1
+tardir="bgs-master"
+desc="simple background setter based on imlib2"
+
+[deps]
+imlib2
+
+[build]
+[ -n "$CROSS_COMPILE" ] && \
+  xconfflags="--host=$($CC -dumpmachine) \
+  --with-sysroot=$butch_root_dir"
```

Comment:
this stanza here is meant to be used with packages using configure scripts. on its own here like that it's just a NOP